src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,45 @@
-import React from 'react';
+import { UserInfo } from '../UserInfo';
+import commentsFromServer from '../../api/comments';
+import usersFromServer from '../../api/users';
+import { Post } from '../../types/post';
+import { CommentList } from '../CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    userId,
+    title,
+    body,
+    id,
+  } = post;
+
+  const user = usersFromServer.find(curUser => curUser.id === userId);
+  const comments = commentsFromServer
+    .filter(curComment => curComment.postId === id);
+
```

Comment:
test for comment you cant to pass because they didn't give in test lucky values ;)
1. they gave on this block new an object wich contains 'user' info and all 'comments'
2. after, you take this values right from .api array  not from parametrs and dont give them chance to get their results which they gave to parametrs. 
the Id they gave is 11 so
you code result gives 5 results
![image](https://user-images.githubusercontent.com/31005050/202580103-e09a1830-12ea-45f8-ad36-fd3e4318f2a1.png)


they wanted to see 3 results. on post id 11.  (throughout commentList ==>> maping commentInfo == >>>)
![image](https://user-images.githubusercontent.com/31005050/202580171-edf347f6-3d06-42e7-a3f2-60e0898d69ab.png)